### PR TITLE
Append columns to first row if data is missing

### PR DESCRIPTION
**Pull request by donnapep**

### CODE REVIEW

**Comment by donnapep** [comment]
@stulees @rodrigopavezi Please review. Thx.

File: rise-google-sheet.html [hunk 1]
```diff
@@ -237,17 +237,49 @@
...
          values (minColumn, maxColumn, minRow, maxRow) are out of scope of the data entered in worksheet
         */
         response.results = (resp.values) ? resp.values : [];
 
+        // Workaround for this issue - https://goo.gl/rsazDP.
+        firstRowCols = (response.results && response.results.length > 0) ? response.results[0].length : 0;
+        totalCols = this._getTotalCols(response.results);
+
+        // Append cells to the first row until it matches the total column count.
+        if (firstRowCols < totalCols) {
```
Comment (stulees) [inline rise-google-sheet.html]: @donnapep There could be also the scenario where not just the first row is missing values, but the initial few rows until there's a row that fills every column. I think you'll have to iterate over each row starting with the first until you hit a row that its length is equal to total cols. Make sense?

Comment (donnapep) [reply rise-google-sheet.html]: @stulees It actually doesn't appear to be necessary. I just tested with several rows of data, all of which have nothing in the last column except the last row, and the last column showed just fine. The Widget likely uses the first row as the defining structure.

Comment (stulees) [reply rise-google-sheet.html]: @donnapep That will suffice for our Widget, but this component gets used elsewhere also (eg. Creative projects). I think we need to fill all empty values to ensure there are no issues for all consumers of the component.

Comment (donnapep) [reply rise-google-sheet.html]: @stulees OK, will adjust. Thx.

**Comment by donnapep** [comment]
@stulees Made changes as per our discussion. Please review. Thx.

**Comment by stulees** [comment]
@donnapep Looks good 👍

**Pull request merged**